src/components/TransactionPreview/HistoryPreview.tsx

```diff
@@ -0,0 +1,177 @@
+  display: inline-block;
+  color: ${({ theme }) => theme.color.text.innerLabel};
+  margin-right: 7px;
+  margin-bottom: 10px;
+  font-size: 14px;
+`;
+
+const ValueWrapper = styled.div<{ marginTop?: number }>`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  ${({ marginTop }) => marginTop && `margin-top: ${marginTop}px;`}
+`;
+
+interface TransactionPreviewInterface {
+  crossChainAction: ICrossChainAction;
+}
+
+const HistoryPreview = ({ crossChainAction }: TransactionPreviewInterface) => {
+  const { accountAddress, providerAddress, web3Provider } = useEtherspot();
+  const theme: Theme = useTheme();
+
+  const { chainId, type } = crossChainAction;
+
+  const openBlockExplorerUrl = (transactionUrl) => {
+    if (!transactionUrl) return;
+    else window.open(transactionUrl, '_blank');
+  };
+
+  const previewSend = (
+    preview: SendAssetActionPreview | null,
+    network: Chain | undefined,
+    chainTitle: string,
+    direction: string
+  ) => {
+    if (!preview) return null;
+
+    const { asset, fromAddress, transactionUrl } = preview;
+    const receiverAddress = preview.receiverAddress as string;
+    const SendReceiveAddress = direction == 'Sender' ? receiverAddress : fromAddress;
+    const gasCostNumericString = ethers.utils.formatUnits(
+      asset.gasCost,
+      nativeAssetPerChainId[crossChainAction.chainId].decimals
+    );
+    const gasAssetSymbol = nativeAssetPerChainId[crossChainAction.chainId].symbol;
+    const gasCostFormatted = `${formatAmountDisplay(gasCostNumericString)} ${gasAssetSymbol}`;
+    const gasFeesUSD = formatAmountDisplay(`${+gasCostNumericString * +crossChainAction.estimated.usdPrice}`, '$');
+
+    const valueToReceiveRaw =
+      asset.feeAmount && asset.feeAmount != null ? ethers.utils.formatUnits(asset?.feeAmount, asset?.decimals) : null;
+
+    const valueToReceive =
+      valueToReceiveRaw != null
+        ? formatAmountDisplay(`${+valueToReceiveRaw * +crossChainAction.estimated.usdPrice}`, '$')
+        : '';
+    const amountString = `${+valueToReceiveRaw + ` ` + asset.symbol}・${valueToReceive}`;
+
+    return (
+      <>
+        <MainWrapper>
+          <LeftWrapper>
+            <Label>{direction == 'Sender' ? ' Send to' : 'Receive from'} &nbsp;</Label>
+            <ClickableText onClick={() => copyToClipboard(receiverAddress)}>
+              {getTypeOfAddress(SendReceiveAddress, accountAddress, providerAddress)}
+            </ClickableText>
+            <ValueWrapper>
+              <CombinedRoundedImages
+                title={asset.symbol}
+                url={asset.iconUrl}
+                smallImageTitle={chainTitle}
+                smallImageUrl={network?.iconUrl}
+                size={24}
+              />
+              <Text> {amountString} </Text>
+            </ValueWrapper>
+          </LeftWrapper>
+          <RightWrapper>
+            <DateLabel>
+              {!!asset.createTimestamp && moment.unix(asset.createTimestamp).format('D.M.YY hh:mm')}
```

Comment:
Is D.M.YY how it is in the design? Just seems like it should be `/`...

Reply:
<img width="80" alt="Screenshot 2023-09-26 at 10 04 42 AM" src="https://github.com/etherspot/etherspot-react-transaction-buidler/assets/141008551/8860516e-de5e-4393-8573-125889885527">
In design, itis D.M.YY

Reply:
changes done as per your suggestion